packages/fx-core/resource/package.nls.json

```diff
@@ -395,6 +398,12 @@
   "core.createProjectQuestion.noPlugin.detail": "Create declarative Copilot only",
   "core.createProjectQuestion.addPlugin.label": "Add plugin",
   "core.createProjectQuestion.addPlugin.detail": "Create declarative Copilot with API plugin",
+  "core.createProjectQuestion.addExistingPlugin.title": "Upload Files",
```

Comment:
Upload files

Reply:
I think we usually do capitalize in title, such as new project :)
![image](https://github.com/user-attachments/assets/1853c9ca-2412-4fe6-9f74-a09fd5fa0ea8)

Reply:
Yes, but that is incorrect. As per MS style guide, we use sentence case.